core/controller/src/main/scala/whisk/core/controller/actions/PrimitiveActions.scala

```diff
@@ -174,23 +175,24 @@ protected[actions] trait PrimitiveActions {
 
     val postedFuture = loadBalancer.publish(action, message)
 
-    postedFuture.flatMap { activeAckResponse =>
-      // successfully posted activation request to the message bus
-      transid.finished(this, startLoadbalancer)
-
+    postedFuture andThen {
+      case Success(_) => transid.finished(this, startLoadbalancer)
+      case Failure(e) => transid.failed(this, startLoadbalancer, e.getMessage, logLevel = ErrorLevel)
```

Comment:
Should the error log level be eliminated here? An activation failure doesn't always imply a system failure

Reply:
Thanks for reminding, we can use the default warn level.

Should this get changed as well?
https://github.com/apache/incubator-openwhisk/blob/0242a60aeb976d070abda76a8ac2aa7e3306e393/core/controller/src/main/scala/whisk/core/controller/Actions.scala#L292

Or I can match exception types for different levels. However, I think logging here is quite redundant b.c. we've already logged this at `doInvoke`.

BTW, in opentracing, there's no standardized warn tag provided, is this fine?

Reply:
I think that using the standard warn level would be fine. 

The snippet you provided is depicting the system error, so the error log level in that place is fine IMO.
 
As long as it doesn't disturb anyone else, I'm fine with leaving the `err` tag for the tracing. My main point was to avoid the log overflood with the messages produced by users, since it could significantly hamper the performance.

Reply:
Ah yes, it's a system error, thanks!